atomic_reactor/plugins/pre_koji_parent.py

```diff
@@ -33,8 +33,8 @@
 
 class KojiParentBuildMissing(ValueError):
     """Expected to find a build for the parent image in koji, did not find it within timeout."""
-    pass
-
+    def __init__(self, error):
```

Comment:
I don't understand how this change relates to the rest of this PR...

Reply:
New pylint screams. However I'm not sure if this fix is correct.

why not just remove `pass` without adding `__init__`?